components/prow/manifests/prow_internals_2.yaml

```diff
@@ -168,7 +168,6 @@ data:
           entrypoint: "registry.svc.ci.openshift.org/ci/entrypoint:latest"
           sidecar: "registry.svc.ci.openshift.org/ci/sidecar:latest"
         gcs_configuration:
-          bucket: "{{ .GCS_BUCKET }}"
           path_strategy: "explicit"
           default_org: "{{ .GITHUB_ORG }}"
           default_repo: "{{ .GITHUB_REPO }}"
```

Comment:
do we need the rest of the gcs config?

Reply:
`hook` & `plank` were crashing when I removed gcs config completely

Reply:
would you mind opening a an issue upstream to explain that it should crash when we don't need a GCS_BUCKET